### PR TITLE
[ci skip] Add 6.0 Release Notes for Action View

### PR DESCRIPTION
**Pull request by alexcameron89**
### Summary

I've added release notes based off of the CHANGELOG for Action View.

### CODE REVIEW

File: guides/source/6_0_release_notes.md [hunk 1]
```diff
@@ -257,10 +257,63 @@ Please refer to the [Changelog][action-view] for detailed changes.
 
 ### Removals
 
+*   Remove deprecated `image_alt` helper.
+    ([Commit](https://github.com/rails/rails/commit/60c8a03c8d1e45e48fcb1055ba4c49ed3d5ff78f))
+
+*   Remove RecordTagHelper.
```
Comment (prathamesh-sonpatki) [inline guides/source/6_0_release_notes.md]: I wonder we should have this as an entry as the gem was already extracted out. If we want to have it, then we can expand a bit ~

```
Remove empty `RecordTagHelper` module from which the functionality was already moved to `record_tag_helper` gem.
```

File: guides/source/6_0_release_notes.md [hunk 2]
```diff
@@ -257,10 +257,63 @@ Please refer to the [Changelog][action-view] for detailed changes.
...
 
+*   Deprecate `ActionView::Template.finalize_compiled_template_methods` with
+    no replacement.
+    ([Pull Request](https://github.com/rails/rails/pull/35036))
+
+*   Deprecate `config.action_view.finalize_compiled_template_methods` with
+    no replacement.
+    ([Pull Request](https://github.com/rails/rails/pull/35036))
+
+*   Deprecate calling private model methods from view helpers.
```
Comment (prathamesh-sonpatki) [inline guides/source/6_0_release_notes.md]: from view helper `options_from_collection_for_select` ?

File: guides/source/6_0_release_notes.md [hunk 3]
```diff
@@ -257,10 +257,63 @@ Please refer to the [Changelog][action-view] for detailed changes.
...
+
+*   Add a `year_format` option to `date_select` tag, making it possible to
+    customize year names.
+    ([Pull Request](https://github.com/rails/rails/pull/32190))
+
+*   Add a `nonce: true` option for `javascript_include_tag` helper to
+    support automatic nonce generation for a Content Security Policy.
+    ([Pull Request](https://github.com/rails/rails/pull/32607))
+
+*   Add a `action_view.finalize_compiled_template_methods` configuration to disable or
```
Comment (prathamesh-sonpatki) [inline guides/source/6_0_release_notes.md]: I don't think we should mention about the default value here. Just the first sentence will suffice.

**Review by prathamesh-sonpatki — CHANGES_REQUESTED** [review]

File: guides/source/6_0_release_notes.md (hunk 1 above)
Comment (alexcameron89) [reply guides/source/6_0_release_notes.md]: I wondered the same, and I like your recommendation, I'll make that change.

**Comment by alexcameron89** [comment]
@prathamesh-sonpatki thank you for your quick and thorough reviews. I've made the recommended changes.

**Review by prathamesh-sonpatki — APPROVED** [review]

**Pull request merged**